spec/phraseapp-in-context-editor-ruby/backend_service_spec.rb

```diff
@@ -71,7 +70,7 @@
       it { is_expected.to eql i18n_translation }
 
       context "given arguments other than key_name" do
-        let(:args) { [key_name, locale: :ru] }
+        subject { phraseapp_service.translate(key_name, locale: :ru) }
```

Comment:
If phrase is disabled the params need to be real keys_word args so I changed all places

Reply:
Note: This is a behaviour change for direct calls to `I18n.t` when this gem is installed but disabled. Right now on master, `I18n.t("foo", {bar: "baz"})` works. With this PR, `I18n.t("foo", {bar: "baz"})` errors.

The new behaviour of this PR matches the behaviour of `t` in views (https://api.rubyonrails.org/classes/ActionView/Helpers/TranslationHelper.html#method-i-translate) both with and with the gem, and `I18n.t` when this gem is not installed - they all [error when a hash is provided](https://github.com/ruby-i18n/i18n/pull/545) instead of keyword argument.

So I think this is a good change to make.

Reply:
Gem disabled
```
>  PhraseApp::InContextEditor.enabled?
 => false
 > I18n.t("foo", { bar: "baz" })
 => "Translation missing: de.foo"
```

Gem not installed
```
> I18n.t("foo", { bar: "baz" })
.../ruby-3.2.3@mhmr/gems/i18n-1.14.4/lib/i18n.rb:210:in `translate': wrong number of arguments (given 2, expected 0..1) (ArgumentError)
```

So this could be a breaking change for some apps and should be released in a major version.

Thanks for the review.